backend-lib/src/main/rust/lib.rs

```diff
@@ -1423,7 +1509,7 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_sh
 
         let shielding_threshold = NonNegativeAmount::from_u64(100000).unwrap();
 
-        zip317_helper(
+        let txid = zip317_helper(
```

Comment:
Comment on the definition of `zip317_helper`:
`fixed::FeeRule::standard()` is deprecated. Consider using `fixed::FeeRule::non_standard(zip317::MINIMUM_FEE)`.

Reply:
Updating the fee model will be part of the this planned issue https://github.com/zcash/zcash-android-wallet-sdk/issues/776